src/main/ipc/skills.copyToAgents.integration.test.ts

```
@@ -0,0 +1,101 @@
+import { mkdir, mkdtemp, readFile, rm, writeFile } from 'node:fs/promises'
+import { join } from 'node:path'
+import { tmpdir } from 'node:os'
+
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
```

ESLint

[error] Unable to resolve path to module 'vitest'.

(import-x/no-unresolved)